create_database.sql

```diff
@@ -0,0 +1,1732 @@
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `system_admin` (
+  `admin_id` int(11) NOT NULL AUTO_INCREMENT,
+  `first_name` varchar(255) DEFAULT NULL,
+  `last_name` varchar(255) DEFAULT NULL,
+  `primary_email` varchar(255) DEFAULT NULL,
+  `password` varchar(255) DEFAULT NULL,
+  `secondary_email` varchar(255) DEFAULT NULL,
+  `phone` varchar(255) DEFAULT NULL,
+  `force_password_reset` int(11) DEFAULT NULL,
+  `status` varchar(255) DEFAULT 'inactive',
+  `is_ptcc_coordinator` int(1) unsigned NOT NULL DEFAULT '0',
+  `created_on` datetime DEFAULT NULL,
+  `created_by` varchar(255) DEFAULT NULL,
+  `updated_on` datetime DEFAULT NULL,
+  `updated_by` varchar(255) DEFAULT NULL,
+  `include_as_pecc_in_reports` int(1) NOT NULL DEFAULT '1',
+  PRIMARY KEY (`admin_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `temp_mail`
+--
+
+DROP TABLE IF EXISTS `temp_mail`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `temp_mail` (
+  `temp_id` int(11) NOT NULL AUTO_INCREMENT,
+  `message` text,
+  `from_mail` varchar(255) DEFAULT NULL,
+  `to_email` varchar(255) NOT NULL,
+  `bcc` text,
+  `cc` text,
+  `subject` text,
+  `from_full_name` varchar(255) DEFAULT NULL,
+  `status` varchar(255) NOT NULL DEFAULT 'pending',
+  PRIMARY KEY (`temp_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `temp_push_notification`
+--
+
+DROP TABLE IF EXISTS `temp_push_notification`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `temp_push_notification` (
+  `temp_id` int(11) NOT NULL AUTO_INCREMENT,
+  `to` varchar(50) NOT NULL,
+  `sound` varchar(20) NOT NULL,
+  `title` varchar(255) NOT NULL,
+  `body` text NOT NULL,
+  `data` longtext,
+  `status` varchar(255) NOT NULL DEFAULT 'pending',
+  PRIMARY KEY (`temp_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping routines for database 'create_database'
+--
+/*!50003 DROP PROCEDURE IF EXISTS `CreateParticipant` */;
+/*!50003 SET @saved_cs_client      = @@character_set_client */ ;
+/*!50003 SET @saved_cs_results     = @@character_set_results */ ;
+/*!50003 SET @saved_col_connection = @@collation_connection */ ;
+/*!50003 SET character_set_client  = utf8 */ ;
+/*!50003 SET character_set_results = utf8 */ ;
+/*!50003 SET collation_connection  = utf8_general_ci */ ;
+/*!50003 SET @saved_sql_mode       = @@sql_mode */ ;
+/*!50003 SET sql_mode              = 'NO_ENGINE_SUBSTITUTION' */ ;
+DELIMITER ;;
+CREATE DEFINER=`root`@`localhost` PROCEDURE `CreateParticipant`(IN countryName VARCHAR(255), IN labName VARCHAR(500), IN labIdentifier VARCHAR(255))
```

Comment:
Please remove `DEFINER=\`root\`@\`localhost\``? The live database that ePT currently run on doesn't have a root user and we want all authorised DB users to be able to edit and execute stored procedures on this database.

Reply:
Fine!